posts/inside-rust/2024-09-23-rtn-call-for-testing.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: "Return Type Notation MVP: Call for Testing!"
```

Comment:
```suggestion
title: "Return type notation MVP: Call for testing!"
```

We've been rather inconsistent about this, but the blog mostly uses sentence-case titles, which has been the trend on the web.  (We might call the capitalization after the colon "piecewise sentence-case".)

Reply:
Sure, I don't care either way.